src/tools/rustbook/build.rs

```diff
@@ -0,0 +1,7 @@
+use std::env::var;
```

Comment:
Does this script need to be declared in the Cargo.toml?

Reply:
Ofc, shame on me.